app/views/welcome_mailer/welcome_notification.text.erb

```diff
@@ -0,0 +1,10 @@
+<%= t("mailer.welcome_mailer.welcome_notification.header") %>
+
+<%= t("mailer.welcome_mailer.welcome_notification.para1") %>
+
+<%= t("mailer.welcome_mailer.welcome_notification.para2", help_url: help_url('') ) %>
+
+<%= t("mailer.welcome_mailer.welcome_notification.para3", feedback_url: feedback_url ) %>
```

Comment:
Since these were long strings I put a `*_html` version in the locale file with <a> tags.